Docs/DocVisao.md

```diff
@@ -0,0 +1,183 @@
+
+### 2.2	Instrução do Problema
+
+A desorganização de partidas de tênis afeta os seus praticantes bem como os proprietários de clubes, ocasionando a falta de incentivo aos jogadores em se desenvolverem dentro do esporte através da prática com adversários distintos e seus diferentes níveis de habilidades. O desenvolvimento de um método gamificado possibilita a solução desse problema, uma vez que proporcia a praticidade dos usuários em encontrar novos oponentes em horários vantajosos para ambos, aliando essas funcionalidades a um sistema de rankings que promove a competividade sadia entre os jogadores e estimula a prática do esporte.
+
+
+### 2.3	Instrução de Posição do Produto
+
+Para os membros dos clubes de tênis que necessitam de incentivo e praticidade pra exercer o esporte, o Nexte é um aplicativo que administra os jogos de tênis e a competitividade entre seus usuários. Comparando-o com o sistema *Quero Jogar Tênis*, o projeto Nexte possui sistema de gamificação que possibilita a imersão do usuário no esporte, possibilitanto também o engajamento das comunidades amadoras já existentes de tênis. O aplicativo divulga o ranking entre jogadores com facilidade e rapidez em relação âs formas anteriores de registro de resultados, como o excel, sites ou aplicativos exclusivos para esse fim.
+
+
+## 3  Descrições da Parte Interessada e do Usuário
+
+Para os proprietários de clubes amadores de tênis, o problema enfrentado é, principalmente, a falta de estímulo dos membros associados, acarretando na falta de frequência e participação nas quadras. O aplicativo visa melhorar o engajamento dos sócios como um incentivador, atuando no aumento o envolvimento dos associados nos eventos, colaborando para a assiduidade.
+
+Para os membros de clubes amadores de tênis, o maior desafio enfrentado é a dificuldade de achar oponentes em horários que se adequem à rotina pessoal. O aplicativo visa administrar o sistema de partidas, atuando principalmente no gerenciamento dos jogos e competições, oferencendo comodidade na organização da prática do esporte.
+
+Para os patrocinadores, que necessitam de um ambiente para expor seus produtos, o aplicativo visa oferecer uma plataforma conveniente para o anúncio de artefatos esportivos, aumentando a visibilidade da marca no nicho propício. 
+
+### 3.1 Demográficos de Mercado
+O aplicativo Nexte pretende atingir as organizações amadoras de tênis, assim como seus sócios. Atualmente, no Brasil, existem cerca de [300 clubes](#14-referencias) que poderiam se beneficiar do aplicativo.
+
+### 3.2 Resumo da Parte Interessada
+A necessidade de desenvolver habilidades na modalidade tênis através do desafio entre oponentes, assim como a utilização de diferentes espaços físicos e o interesse em organizar as partidas e seus respectivos rankings, apresentam três partes interessadas: tenistas, clubes e desenvolvedores do projeto.
```

Comment:
Acho que a resposta está boa, porém talvez seria interessante trazer as partes interessadas para o início da resposta. O título do tópico não parece ter ligação com a resposta até chegar ao final da leitura.